packages/ripple-nuxt-tide/modules/landing-page/tide.config.js

```diff
@@ -264,10 +279,10 @@ module.exports = {
       'paragraph--card_promotion_auto': {
         component: 'rpl-card-promotion',
         props: {
-          'image': [
-            ['field_paragraph_reference', 'field_featured_image', 'field_media_image', 'url'],
-            ['field_paragraph_reference', 'field_media_image', 'url']
-          ],
+          'image': {
+            field: ['field_paragraph_reference'],
+            filters: ['cardAutoImage']
```

Comment:
Hey @dylankelly If a custom site has it's own custom card mapping, will this PR add a breaking change for them? Will custom card image continually working?

Reply:
@tim-yao no change hopefully, in https://github.com/dpc-sdp/ripple/pull/721/files#diff-9fc14280ef524d93415101f9ed90ce1eR60 we check if image is a string and map it appropriately